MbDotNet/Models/Stubs/HttpStub.cs

```diff
@@ -123,13 +124,36 @@ public HttpStub Returns(HttpStatusCode statusCode, IDictionary<string, string> h
 
             var response = new IsResponse<HttpResponseFields>(fields);
 
+            return Returns(response);
+        }
+
+        /// <summary>
+        /// Adds a proxy to the stub with the specified content type
+        /// </summary>
+        /// <param name="to">endpoint address to proxy to</param>
+        /// <param name="proxyMode">proxyalways, proxyonce or proxytransparent</param>
+        /// <param name="predicateGenerators">list of predicates that a proxy repsonse will be recorded for</param>
+        /// <returns></returns>
+        public HttpStub Returns(Uri to, ProxyMode proxyMode, List<PredicateBase> predicateGenerators)
```

Comment:
Maybe change this method to `ReturnsProxy` or something similar to differentiate it from the `Returns` methods that will give back specific content.

Something similar should also be implemented on the `TcpStub`.